lib/generators/geoblacklight/templates/catalog_controller.rb

```diff
@@ -85,38 +84,36 @@ class CatalogController < ApplicationController
     #    :years_25 => { :label => 'within 25 Years', :fq => "pub_date:[#{Time.now.year - 25 } TO *]" }
     # }
 
-    config.add_facet_field Settings.FIELDS.PROVIDER, label: 'Institution', limit: 8, partial: "icon_facet"
-    config.add_facet_field Settings.FIELDS.CREATOR, :label => 'Author', :limit => 8
-    config.add_facet_field Settings.FIELDS.PUBLISHER, :label => 'Publisher', :limit => 8
-    config.add_facet_field Settings.FIELDS.SUBJECT, :label => 'Subject', :limit => 8
-    config.add_facet_field Settings.FIELDS.SPATIAL_COVERAGE, :label => 'Place', :limit => 8
-    config.add_facet_field Settings.FIELDS.IS_PART_OF, :label => 'Collection', :limit => 8
 
-    config.add_facet_field Settings.FIELDS.INDEX_YEAR, :label => 'Year', :limit => 10
+#FACETS
 
-    config.add_facet_field Settings.FIELDS.RIGHTS, label: 'Access', limit: 8, partial: "icon_facet"
-    config.add_facet_field Settings.FIELDS.RESOURCE_TYPE, label: 'Data type', limit: 8, partial: "icon_facet"
+	#DEFAULT FACETS
+	#to add additional facets, use the keys defined in the settings.yml file
+    config.add_facet_field Settings.FIELDS.RESOURCE_CLASS, label: 'Resource Class', :limit => 8    
+    config.add_facet_field Settings.FIELDS.ACCESS_RIGHTS, label: 'Access', limit: 8, partial: "icon_facet"
+    config.add_facet_field Settings.FIELDS.INDEX_YEAR, :label => 'Year', :limit => 10
+    config.add_facet_field Settings.FIELDS.SPATIAL_COVERAGE, :label => 'Place', :limit => 8
+    config.add_facet_field Settings.FIELDS.CREATOR, :label => 'Creator(s)', :limit => 8
+    config.add_facet_field Settings.FIELDS.PUBLISHER, :label => 'Publisher', :limit => 8
+    config.add_facet_field Settings.FIELDS.PROVIDER, label: 'Provider', limit: 8, partial: "icon_facet"
     config.add_facet_field Settings.FIELDS.FORMAT, :label => 'Format', :limit => 8
-    config.add_facet_field Settings.FIELDS.SOURCE, show: false
+    config.add_facet_field Settings.FIELDS.RESOURCE_TYPE, label: 'Resource Type', :limit => 8
+    config.add_facet_field Settings.FIELDS.SUBJECT, :label => 'Subject', :limit => 8
+    config.add_facet_field Settings.FIELDS.ISO_TOPIC_CATEGORY, :label => 'Theme', :limit => 8
+    config.add_facet_field Settings.FIELDS.SOURCE, :label => 'Collection', :limit => 8
```

Comment:
change to:
`config.add_facet_field Settings.FIELDS.SOURCE, :label => 'Collection', :limit => 8, :show => false`

Reply:
Thanks. That worked!